dotnet/src/Connectors/Connectors.AI.OpenAI/OpenAIKernelBuilderExtensions.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.SemanticKernel;
 /// <summary>
 /// Provides extension methods for the <see cref="KernelBuilder"/> class to configure OpenAI and AzureOpenAI connectors.
 /// </summary>
-public static class OpenAIKernelBuilderExtensions
+public static class OpenAKernelBuilderExtensions
```

Comment:
I don't think you meant to make this change

Reply:
Yep, right, it was not intentional change. Thanks.